### PR TITLE
Added info about chromedriver

### PR DESCRIPTION
**Pull request by lzap**
This solves the error I see when running discovery tests.

### CODE REVIEW

File: contribute.md [hunk 1]
```diff
@@ -77,16 +77,21 @@ for RHEL/Fedora based distributions).
 
 You can also exclude these features by using `bundle install --without libvirt postgresql` etc (groups are under bundler.d/).
 
+To run integration tests, it's best to install chromedriver package from your
+distribution as it comes with compatible chromium headless browser. For example
+in Fedora do this: `yum install chromedriver`. If you don't do this, npm will
+install chromedriver that might be not compatible with your Chrome or Chromium
+browser which can lead to error "Chrome version must be between XX and YY."
+
+When using system chromedriver, set the correct path to it via vabieble. For
```
Comment (ekohl) [inline contribute.md]:
```suggestion
When using system chromedriver, set the correct path to it via variable. For
```

Comment (ezr-ondrej) [inline contribute.md]:
```suggestion
When using system chromedriver, set the correct path to it via variable. For
```

**Comment by lzap** [comment]
Thanks, please squash on merge!

**Comment by ekohl** [comment]
Thanks!

**Pull request merged**